packages/appium/lib/appium.js

```diff
@@ -201,10 +201,30 @@ class AppiumDriver extends DriverCore {
       'Please check https://github.com/appium/appium/issues/20880 for more details.');
   }
 
-  async getAppiumSessionCapabilities () {
-    throw new errors.NotImplementedError('TODO: Implement this method body to call getAppiumSessionCapabilities ' +
-      'from the destination driver instance in a session.'
-    );
+
+  /**
+   * Return session capabilities info related to the destination driver.
+   * https://github.com/appium/appium/issues/20880
+   *
+   * @param sessionId - the id of the session to return capabilities info.
+   * @returns {Promise<{value: object, error?: Error, protocol?: string}>}
+   */
+  async getAppiumSessionCapabilities (sessionId) {
```

Comment:
after looking at this implementation for the second time it does not really make sense to have this method in the umbrella driver since the only purpose of it is to call the corresponding driver-specific implementation

Reply:
Yea, i was confused with the same thing. I think right now this endpoint can return an error message to refer to https://github.com/appium/appium/issues/20880 until we'll define concrete return values.